capstone/src/test/java/com/google/univiz/api/SearchResourceImplTest.java

```diff
@@ -0,0 +1,76 @@
+package com.google.univiz.api;
+
+import static com.google.common.truth.Truth.assertThat;
+import static org.mockito.Mockito.when;
+
+import com.google.common.collect.Lists;
+import com.google.univiz.api.representation.CollegeId;
+import com.google.univiz.api.representation.SearchData;
+import com.google.univiz.api.representation.SuggestionData;
+import com.google.univiz.api.representation.SuggestionResponse;
+import java.util.List;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+import org.mockito.Mock;
+import org.mockito.junit.MockitoJUnit;
+import org.mockito.junit.MockitoRule;
+
+@RunWith(JUnit4.class)
+public final class SearchResourceImplTest {
+  @Mock private SuggestionDataApi mockSuggestionApi;
+  @Mock private SuggestionResponse cannedResponse;
+  @Mock private SuggestionData cannedSuggestion;
+  @Rule public final MockitoRule mockitoRule = MockitoJUnit.rule();
+  private SearchResourceImpl search;
+
+  @Before
+  public void setup() {
+    search = new SearchResourceImpl(mockSuggestionApi);
+  }
+
+  @Test
+  public void testSuggestionsForCollegeNameNoSpace() throws Exception {
+    String collegeName = "Stanford";
+    SearchResourceImpl search = new SearchResourceImpl(mockSuggestionApi);
```

Comment:
Isn't `search` already initialized in your `setup` method?

Reply:
yes indeed-removed!